packages/arb-bridge-peripherals/contracts/tokenbridge/ethereum/gateway/L1CustomGateway.sol

```diff
@@ -81,6 +82,29 @@ contract L1CustomGateway is L1ArbitrumExtendedGateway, ICustomGateway {
             );
     }
 
+    function outboundTransferCustomRefundWithPermit(
```

Comment:
no need to override here if we are not doing anything special

Reply:
when I take out override in the gateways, the compiler throws an error "Overriding function is missing "override" specifier"

Reply:
I mean we can just remove the whole thing